pinot-query-planner/src/test/resources/queries/GroupByPlans.json

```diff
@@ -91,6 +91,92 @@
           "\n                  LogicalTableScan(table=[[a]])",
           "\n"
         ]
+      },
+      {
+        "description": "SQL hint based group by optimization with select and aggregate column",
+        "sql": "EXPLAIN PLAN FOR SELECT /*+ SKIP_LEAF_STAGE_GROUP_BY_AGGREGATION */ a.col1, SUM(a.col3) FROM a GROUP BY a.col1",
```

Comment:
Can you add a separate test for GROUP BY with `AVG()` as well ?

Reply:
Added.